example/extra-resources-present/functions.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: pkg.crossplane.io/v1
+kind: Function
+metadata:
+  name: fn-hcl
+spec:
+  package: xpkg.crossplane.io/crossplane-contrib/function-hcl:v0.0.2
```

Comment:
Is there a `latest` tag that could be used? Running the example outside the development environment will fail with this version